hsmetrics.wdl

```diff
@@ -0,0 +1,160 @@
+version 1.0
+
+workflow hsMetrics {
+input {
+    File    inputBam
+    File    baitBed
+    File    targetBed
+    String  refFasta
+    String? outputPrefix = ""
+    String? stringencyFilter = "LENIENT"
+    Float?  minPct      = 0.5
+    Int?    coverageCap = 500
+}
+
+call makeRefDictionary { input: refFasta = refFasta }
+call bedToIntervals as bedToTargetIntervals { input: inputBed = targetBed, refDict = makeRefDictionary.refDict }
+call bedToIntervals as bedToBaitIntervals { input: inputBed = baitBed, refDict = makeRefDictionary.refDict }
+
+call collectHSmetrics{ input: inputBam = inputBam, baitIntervals = bedToBaitIntervals.outputIntervals, targetIntervals = bedToTargetIntervals.outputIntervals, refFasta = refFasta, filter = stringencyFilter, coverageCap = coverageCap, outputPrefix = outputPrefix }
+call collectInsertMetrics{ input: inputBam = inputBam, minPct = minPct, outputPrefix = outputPrefix }
+
+output {
+  File outputHSMetrics  = collectHSmetrics.outputHSMetrics
+  File outputINSMetrics = collectInsertMetrics.outputINSMetrics
+  File outputINSPDF     = collectInsertMetrics.outputINSPDF
+}
+
+}
+
+# ==========================================
+#  TASK 1 of 4: make reference dictionary
+# ==========================================
+task makeRefDictionary {
+input {
+        String refFasta
+        Int?   jobMemory  = 18
+        Int?   javaMemory = 10
```

Comment:
Can this be modified to one memory parameter and then something like `~{jobMemory -4}` in the command block

Reply:
implemented